example/model/augmentations.json

```diff
@@ -0,0 +1,3 @@
+[
+    "localization"
+]
```

Comment:
This is an example specific to local-orbital DFT only. Let's omit it for now. We can put specific examples in their corresponding folders later on.